Emby.Server.Implementations/ScheduledTasks/Tasks/ChapterImagesTask.cs

```diff
@@ -115,7 +115,8 @@ public async Task ExecuteAsync(IProgress<double> progress, CancellationToken can
             {
                 try
                 {
-                    previouslyFailedImages = File.ReadAllText(failHistoryPath)
+                    var failHistoryText = await File.ReadAllTextAsync(failHistoryPath, cancellationToken).ConfigureAwait(false);
+                    previouslyFailedImages = failHistoryText
```

Comment:
Why in 2 parts?

Reply:
@Bond-009, it looks a bit cumbersome. Let's fix it if you don't think so
```
previouslyFailedImages = (await File.ReadAllTextAsync(failHistoryPath, cancellationToken).ConfigureAwait(false))
    .Split('|', StringSplitOptions.RemoveEmptyEntries)
    .ToList();
```

Reply:
@Bond-009, and could you check my questions above?

Reply:
I didn't notice the split below, I'd move that to the same line